### PR TITLE
add(clickhouse): total storage limitation

### PR DESCRIPTION
**Pull request by wojcik-dorota**
https://aiven.atlassian.net/browse/DDB-1361

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

File: static/includes/clickhouse-storage-limitation.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+:::note[Total storage per plan]
+In service plans with a single shard, storage capacity doesn't depend on the number of
+nodes, and the data of this single shard is replicated on all the available nodes.
+
+For example, with a Startup-16 plan you get 1150 GB of total storage per VM, same as with
```
Comment (Khatskevich) [inline static/includes/clickhouse-storage-limitation.md:5]: That is a good explanation.
I would also add a small clarification for the sharded plan here.

Comment (wojcik-dorota) [reply static/includes/clickhouse-storage-limitation.md:5]: Thank you @Khatskevich ! What would you specifically include for the sharded plan?

Comment (Khatskevich) [reply static/includes/clickhouse-storage-limitation.md:5]: May be give another example: For example we have Premium-6x-16 which has 2 shards and 6 servers.
This means that there are 6 servers with 1150GB storage each and the data is replicated 3 times.

And the conclusion: Total storage is the amount of data which may be inserted to the service, which will be replicated `number of nodes` / `number of shards` times.

**Review by Khatskevich — APPROVED** [review]

**Review by harshini-rangaswamy — APPROVED** [review]
Look good!

**Pull request merged**